_posts/pl/2021-03-xx-wspoldzielona-biblioteka-w-jenkins-pipeline.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
Przyznam, że niezbyt podoba mi się to, ze mamy niepełny przykład. Przechodząc to co jest w tym artykule nie dostaniemy działającego pipeline'a, bo nie ma VersionBumpera

Reply:
Może i tak tylko że kod robiący bumpa niepotrzebnie zaśmieci clou libów. BTW jest duży :/

Reply:
A ja nie mam z tym problemu :) Nawet widząc komentarze chciałem wpaść i zaproponować, że nie ma potrzeby wrzucać kodu, który nie jest sensowny z punktu widzenia wpisu.

Czy dajemy link do gotowego repo? Jeśli tak to nie ma to żadnego znaczenia.

Reply:
A i przyznam, że jeszcze nie widziałem całości, stąd pytanie: czy to jest wpis tytu tutorial (pokazujący jak coś zrobić) czy raczej guide (pokazujący jakiś problem)?

Reply:
Chciałem żeby bardziej była to wędka niż marchewka :/ stąd pewne uproszczenia. Wpis ma pokazywać że jest taka możliwość nie do końca rozwiązywać konkretny problem. Jest on oparty na przykładzie, ale żeby pokazać możliwość zastosowania.

Reply:
Dla mnie to jest ok, takie też miałem poczucie czytając całość. Stąd, dla mnie jest ok żeby nie było dokładnych kodów :)